pwsh/get-vcpkg-deps.ps1

```diff
@@ -26,18 +26,20 @@ if ($IsWindows) {
     sudo apt-get install nasm libxi-dev libgl1-mesa-dev libglu1-mesa-dev mesa-common-dev libxrandr-dev libxxf86vm-dev
 }
 
-# Set up prefixes
+# Set default triplet
 if ($IsWindows) {
-    & "$env:GITHUB_WORKSPACE\pwsh\vcvars.ps1"
```

Comment:
Removed this as it appears to be redundant. Line 18 in this file has already run it.